compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirMemberPropertyChecker.kt

```diff
@@ -71,14 +71,14 @@ object FirMemberPropertyChecker : FirRegularClassChecker() {
                 reporter.reportOn(it, FirErrors.ABSTRACT_DELEGATED_PROPERTY, context)
             }
 
-            checkAccessor(property.getter, property.delegate) { src, _ ->
-                reporter.reportOn(src, FirErrors.ABSTRACT_PROPERTY_WITH_GETTER, context)
+            checkAccessor(property.getter, property.delegate) { src, _, hasCustomBody ->
```

Comment:
Maybe personal preference, but having `custom` sounds awkward. Just `hasBody`? If it's a default accessor, it won't have a body anyway.